src/Hangfire.Core/BackgroundJobServer.cs

```diff
@@ -163,5 +174,47 @@ public void Start()
         public void Stop()
         {
         }
+
+        public void OnNext(int i)
+        {
+            int threshold = int.Parse(ConfigurationManager.AppSettings["failedJobCountThreshold"]);
```

Comment:
use var please and also you should switch from int.Parse to int.TryParse there is a chance the data from appsettings could be wrong